drivers/d3d12/rendering_device_driver_d3d12.h

```diff
@@ -395,6 +397,33 @@ class RenderingDeviceDriverD3D12 : public RenderingDeviceDriver {
 	struct VertexFormatInfo {
 		TightLocalVector<D3D12_INPUT_ELEMENT_DESC> input_elem_descs;
 		TightLocalVector<UINT> vertex_buffer_strides;
+
+		Vector<uint8_t> serialize() const {
+			#define serialize_field(x)    \
+				memcpy(p, &x, sizeof(x)); \
+				p += sizeof(x);
+
+			Vector<uint8_t> out;
+			size_t sz = vertex_buffer_strides.size() * sizeof(UINT) + input_elem_descs.size() * (sizeof(D3D12_INPUT_ELEMENT_DESC) - sizeof(LPCSTR));
+			for (auto ed : input_elem_descs) {
```

Comment:
`auto` is [not allowed](https://contributing.godotengine.org/en/latest/engine/guidelines/cpp_usage_guidelines.html#auto-keyword)